src/ovirtapi/index.js

```diff
@@ -303,17 +306,16 @@ const OvirtApi = {
     return httpGet({ url: `${AppConfiguration.applicationContext}/api/storagedomains/${storageId}/files` })
   },
 
-  getCDRom ({ vmId, running }: { vmId: string, running: boolean }): Promise<Object> {
+  getCDRom ({ vmId, current = true }: { vmId: string, current?: boolean }): Promise<ApiCdRomType> {
```

Comment:
The API still supports it, so it could be used, it just isn't right now.  If someone in the future does want to be able to fetch/change the offline CD, they could.  I don't see any harm in leaving the parameter available but defaulted.

Reply:
My rethink makes use of the current option.  :-)